pkg/apis/defaults/configmaps.go

```diff
@@ -9,4 +9,8 @@ const (
 	RadixGitChangedFolders = "git-changed-folders"
 	// RadixGitChangedChangedRadixConfig Indicates if Radix config file is changed in the GitHub source
 	RadixGitChangedChangedRadixConfig = "git-changed-radixconfig"
+	GitPublicKeyConfigMapName         = "git-ssh-public-key"
+	GitPublicKeyConfigMapKey          = "public-key"
+	GitPrivateKeySecretName           = "git-ssh-keys"
+	GitPrivateKeySecretKey            = "id-rsa"
```

Comment:
```suggestion
	GitPrivateKeySecretKey            = "id_rsa"
```

The existing key is "id_rsa" (underscore). GitPrivateKeySecretKey must use the same key name to prevent regeneration of all keys.